deployment/deployment.json

```diff
@@ -9758,5 +9758,35 @@
         }
       }
     }
+  },
+  "friend-tech": {
+    "schema": "generic",
+    "base": "friend-tech",
+    "protocol": "friend-tech",
+    "project": "friend-tech",
+    "deployments": {
+      "friend-tech-base": {
+        "network": "base",
+        "status": "dev",
+        "versions": {
+          "schema": "1.0.0",
```

Comment:
change to 2.1.1 (whatever the generic schema is), otherwise it messes up our ingestion